index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio-Jorum Odiemo</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="Top">
```

Comment:
- [ ] Please try as much as possible to make your `header`design looks like the original in Figma. I suggest that adding a margin can fix the issue. 
- Your Design 
![3](https://user-images.githubusercontent.com/63972670/171396848-e6431334-bc40-4ef1-b546-858abf9cf4e4.PNG)
- Original
![2](https://user-images.githubusercontent.com/63972670/171397058-9a7be016-fe61-48ef-be4f-1bcfd771253a.PNG)